prepline_oer/api/raters.py

```diff
@@ -41,12 +39,54 @@
 
 # pipeline-api
 
+field_locs = [
+    {
+        "rated": [20.0, 76.0, 253.0, 100.0],
+        "rated_position": [20.0, 453.0, 305.0, 473.0],
+        "rater": [20.0, 169.0, 302.0, 193.0],
+        "rater_position": [459.0, 169.0, 591.0, 193.0],
+        "intermediate": [20.0, 219.0, 302.0, 243.0],
+        "intermediate_position": [459.0, 219.0, 591.0, 243.0],
+        "senior": [20.0, 269.0, 253.0, 293.0],
+        "senior_position": [459.0, 269.0, 591.0, 293.0],
+        "duty_description": [20.0, 475.0, 592.0, 565.0],
+        "rater_comments": [20.0, 704.0, 592.0, 753.0],
+    },
+    {
+        "character": [165.0, 56.0, 592.0, 107.0],
+        "presence": [165.0, 108.0, 592.0, 159.0],
+        "intellect": [165.0, 160.0, 592.0, 211.0],
+        "leads": [165.0, 212.0, 592.0, 263.0],
+        "develops": [165.0, 264.0, 592.0, 315.0],
+        "achieves": [165.0, 316.0, 592.0, 367.0],
+        "intermediate_rater_comments": [20.0, 379.0, 592.0, 513.0],
+        "senior_rater_comments": [175.0, 543.0, 592.0, 700.0],
+        "next_assignments": [175.0, 701.0, 592.0, 753.0],
+    },
+]
+
 
-def get_layout(filename="", file=None, model=None):
+def get_textblock(name, loc):
+    block = TextBlock(Rectangle(*loc), type=name)
+    return block
+
+
+def get_fixed_layout(loc_d):
```

Comment:
Could you add a docstring to this function? Especially since `loc_d` ins't self explanatory.